src/renderer/styles/MainContent.scss

```diff
@@ -28,4 +28,13 @@ $band-width: 64px;
     width: $band-width;
     gap: $spacing-s;
   }
+
+  .sliderButtons {
+    height: $band-height;
+    justify-content: center;
+    div {
+      margin-top: 5px;
```

Comment:
minor thing but could we try and stick to using the `_spacing.scss` values for spaces? there should be one for `4px` that could work here